src/libc/absf.s

```diff
@@ -0,0 +1,14 @@
+#include "ultra64/asm.h"
+
+#if OOT_DEBUG
+.set noreorder
+#endif
+
+.section .text
+
+.balign 16
+
+LEAF(absf)
+    abs.s       $f0, $f12
+    jr          $ra
+END(absf)
```

Comment:
```suggestion
LEAF(absf)
    abs.s       $f0, $f12
    jr          $ra
#if OOT_DEBUG
     nop
#endif
END(absf)
```
We actually need to add an explicit nop in the delay slot when noreorder is enabled, since otherwise this file assembles to (noting that we pass `--no-pad-sections` to gas)
```
Disassembly of section .text:

00000000 <absf>:
   0:   46006005        abs.s   $f0,$f12
   4:   03e00008        jr      ra
```
It only matches because the linker will pad files to 0x10, inserting two words of 0 that happens to behave like a nop.

It might also work to just add a nop without wrapping it in if, it should match regardless

Reply:
Seems to work without the if, thanks. Man branch delay slots were such a huge mistake, what a short-sighted decision that was